lib/translate/common.h

```diff
@@ -15,10 +15,16 @@
 #ifndef LIB_TRANSLATOR_COMMON_H
 #define LIB_TRANSLATOR_COMMON_H
 
+namespace translator {
+
+enum class TranslatorStatusCode { SUCCESS, FAILURE };
```

Comment:
Also, i think enum names should the kName format https://google.github.io/styleguide/cppguide.html#Enumerator_Names

Reply:
Ahh, thanks for pointing that out, totally forgot!